fhf-datafangst-client/src/components/Benchmark/Graphs/SpeciesHistogram.tsx

```diff
@@ -45,10 +46,15 @@ export const SpeciesHistogram: FC = () => {
   const species = useAppSelector(selectSpeciesFiskeridir);
   const numHistoric = useAppSelector(selectBenchmarkNumHistoric);
   const selectedDatasource = useAppSelector(selectBenchmarkDataSource);
+  const followTrips = useAppSelector(selectBenchmarkTrips);
+  const followTripsList = Object.values(followTrips).reduce(
```

Comment:
const followTripsList = Object.values(followTrips).flat(); will produce the same result, no?